pgml-dashboard/static/css/scss/components/_forms.scss

```diff
@@ -257,6 +257,19 @@
     caret-color: #{$input-color};
 }
 
+.form-control {
+    &.is-invalid {
+        &:focus {
+            box-shadow: none;
+            border-width: 2px;
+        }
+    }
+}
+
+.invalid-feedback {
+    --bs-danger-text: #{$peach-shade-100};
```

Comment:
As of now all the error colors should be the same and use the variable $error set to $peach-shade-100.  So, it might be best to use $error and deviate later if needed.  Same comment for the above $form-feedback-invalid-color: